tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -65,6 +65,7 @@ limitations under the License.
 #include "tensorflow/lite/kernels/internal/strided_slice_logic.h"
 #include "tensorflow/lite/kernels/internal/tensor.h"
 #include "tensorflow/lite/kernels/internal/types.h"
+#include "third_party/eigen3/Eigen/Core"
```

Comment:
i'm assuming clang-format made this change?

Reply:
yes